api/iterator.go

```diff
@@ -0,0 +1,67 @@
+package api
+
+import (
+	dbm "github.com/tendermint/tm-db"
+	"sync"
+)
+
+// frame stores all Iterators for one contract
+type frame []dbm.Iterator
+
+// iteratorStack contains one frame for each contract, indexed by a counter
+var iteratorStack = make(map[uint64]frame, 60)
```

Comment:
why did you set the initial capacity to 60? This implies that contracts will regularly perform ~60 callbacks, doesn't it?

Reply:
Totally arbitrary. It's not just depth, it also includes queries running in parallel. 

Yeah, I assume depth 3 for the one running contract and depth 3 for the query should be more than enough. I can lower to 10. But reserving <1KB for a map doesn't really have any impact, I can also not reserve any space....